src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -667,6 +682,18 @@ function CreatePipelineRun(props) {
               setState(state => ({ ...state, timeout: value }))
             }
           />
+          <Toggle
+            defaultToggled={false}
+            id="pending-pipeline-toggle"
+            labelText={intl.formatMessage({
+              id: 'dashboard.createPipelineRun.pendingPipilinRun',
+              defaultMessage: 'Create PipelineRun in pending state'
+            })}
+            onChange={handleToggleChange}
```

Comment:
`onToggle` seems to give a cleaner interface here, providing the value of `checked` as the first param so we don't have to deal with the event object at all.